src/pages/Impressum.tsx

```diff
@@ -0,0 +1,135 @@
+          Zeitpunkt der Linksetzung frei von rechtswidrigen Inhalten. Eine
+          ständige inhaltliche Überprüfung der externen Links ist ohne konkrete
+          Anhaltspunkte einer Rechtsverletzung nicht möglich. Bei direkten oder
+          indirekten Verlinkungen auf die Webseiten Dritter, die außerhalb
+          unseres Verantwortungsbereichs liegen, würde eine
+          Haftungsverpflichtung ausschließlich in dem Fall nur bestehen, wenn
+          wir von den Inhalten Kenntnis erlangen und es uns technisch möglich
+          und zumutbar wäre, die Nutzung im Falle rechtswidriger Inhalte zu
+          verhindern. Diese Haftungsausschlusserklärung gilt auch innerhalb des
+          eigenen Internetauftrittes „KiWa“ gesetzten Links und Verweise von
+          Fragestellern, Blogeinträgern, Gästen des Diskussionsforums. Für
+          illegale, fehlerhafte oder unvollständige Inhalte und insbesondere für
+          Schäden, die aus der Nutzung oder Nichtnutzung solcherart
+          dargestellten Informationen entstehen, haftet allein der
+          Diensteanbieter der Seite, auf welche verwiesen wurde, nicht
+          derjenige, der über Links auf die jeweilige Veröffentlichung lediglich
+          verweist. Werden uns Rechtsverletzungen bekannt, werden die externen
+          Links durch uns unverzüglich entfernt.
+        </p>
+      </div>
+      <div>
+        <h2>Urheberrecht</h2>
+        <p>
+          Die auf unserer Webseite veröffentlichen Inhalte und Werke unterliegen
+          dem deutschen Urheberrecht
+          (http://www.gesetze-im-internet.de/bundesrecht/urhg/gesamt.pdf). Die
+          Vervielfältigung, Bearbeitung, Verbreitung und jede Art der Verwertung
+          des geistigen Eigentums in ideeller und materieller Sicht des Urhebers
+          außerhalb der Grenzen des Urheberrechtes bedürfen der vorherigen
+          schriftlichen Zustimmung des jeweiligen Urhebers i.S.d.
+          Urhebergesetzes
+          (http://www.gesetze-im-internet.de/bundesrecht/urhg/gesamt.pdf).
+          Downloads und Kopien dieser Seite sind nur für den privaten und nicht
+          kommerziellen Gebrauch erlaubt. Sind die Inhalte auf unserer Webseite
+          nicht von uns erstellt wurden, sind die Urheberrechte Dritter zu
+          beachten. Die Inhalte Dritter werden als solche kenntlich gemacht.
+          Sollten Sie trotzdem auf eine Urheberrechtsverletzung aufmerksam
+          werden, bitten wir um einen entsprechenden Hinweis. Bei Bekanntwerden
+          von Rechtsverletzungen werden wir derartige Inhalte unverzüglich
+          entfernen.Dieses Impressum wurde freundlicherweise von jurarat.de zur
+          Verfügung gestellt.
+        </p>
+      </div>
+    </>
+  );
+};
+
+class Impressum extends React.PureComponent {
+  render() {
+    return (
+      <div style={{ paddingTop: 50 }}>
+        <h2>Impressum</h2>
+        <Haftung />
+      </div>
+    );
+  }
+}
```

Comment:
Ist schonmal ganz gut. Du benutzt oben eine Functional react component und unten eine Class component. Würde daher vorschlagen unten auch ne Functional draus zu machen.
```suggestion
const Impressum = () => {
    return (
      <div style={{ paddingTop: 50 }}>
        <h2>Impressum</h2>
        <Haftung />
      </div>
    );
  }
```